sources_test.go

```diff
@@ -254,3 +254,49 @@ func TestStdin(t *testing.T) {
 		t.Errorf("want %q, got %q", want, string(got))
 	}
 }
+
+func TestPrompt(t *testing.T) {
+	tcs := []struct {
+		Name        string
+		Input       string
+		Default     string
+		Want        string
+		ErrExpected bool
+	}{
+		{"User Input OK", "123", "notused", "123", false},
```

Comment:
It's not immediately clear with field-less literals like this which field is which. Let's write these as multi-line literals with field names, to make it clear.

Reply:
Done